python-threatexchange/threatexchange/signal_type/index.py

```diff
@@ -48,6 +48,19 @@ def __init__(self, distance: int, metadata: T) -> None:
         self.metadata = metadata
 
 
+class VPDQIndexMatch(IndexMatch):
+    query_match_percent: float
+    compared_match_percent: float
```

Comment:
(not sure): You might be able to leave these out, and the typechecker can figure it out from reading the init, but I don't know